### ISSUE TITLE
stbexceptions::client_error (004) -> 'NoneType' object is not subscriptable

### ISSUE THREAD

**Issue by Juliandev02** [Strawberry-Foundations/strawberry-chat#93]

**Comment by Juliandev02** [comment]
```python
Traceback (most recent call last):
  File "/home/julian/strawberry-chat/server/server.py", line 236, in client_thread
    if isMuted(user):
       ^^^^^^^^^^^^^
  File "/home/julian/strawberry-chat/server/src/functions.py", line 275, in isMuted
    if mutedStatus[0] == "true": return True
       ~~~~~~~~~~~^^^
TypeError: 'NoneType' object is not subscriptable
```

**Comment by Juliandev02** [comment]
Fixed with b371ac83ce403b1946b34418dd95da6d5d89912f & 3388a4dc6d1a5313d66c56113f0052e194882a46
